spec/requests/articles_spec.rb

```diff
@@ -1,11 +1,78 @@
 require 'rails_helper'
 
 RSpec.describe "Articles", type: :request do
+
+  before(:all) do
+    FactoryBot.create(:article)
+  end
+
   describe "GET article index" do
-    it "returns http success" do
-      get "/articles#index"
+    it "returns success with the articles index template" do
+      get articles_path
+
+      expect(response).to have_rendered(:index)
       expect(response).to have_http_status(:success)
     end
   end
 
+  describe "GET /articles/:id" do
+    it "returns success with the articles index template" do
+      get article_path(1)
+
+      expect(response).to have_rendered(:show)
+      expect(response).to have_http_status(:success)
+    end
+  end
+
+  describe "GET /articles/new" do
+    it "returns success with the articles/new template" do
+      get new_article_path
+
+      expect(response).to have_rendered(:new)
+      expect(response).to have_http_status(:success)
+    end
+  end
+
+  describe "POST /articles" do
+    context "with valid parameters" do
+      it "successfully creates an article" do
+        post articles_path, :params => {:article => {:title => Faker::Hipster.sentence(word_count: 3), :body => Faker::Hipster.paragraph}}
+
+        expect(response).to have_http_status(:found)
+      end
+    end
+
+    context "with invalid parameters" do
+      it "responds with unprocessable entity" do
+        post articles_path, :params => {:article => {:title => "", :body => "Foo"}}
```

Comment:
I don't think it is very common to use this hash rocket syntax instead of the JSON style.